### PR TITLE
Improve coverage for dataframe.py

### PR DESCRIPTION
**Pull request by V1NAY8**
For #327 

This PR will increase the coverage by reducing the misses from 309 to 295. 

@sethmlarson Take a look :)

### CODE REVIEW

**Comment by sethmlarson** [comment]
jenkins test this please

File: tests/dataframe/test_drop_pytest.py [hunk 1]
```diff
@@ -50,3 +57,30 @@ def test_drop_all_index(self, df):
         ):
             assert dropped.shape == (0, cols)
             assert list(dropped.to_pandas().index) == []
+
+    def test_drop_raises(self):
+        ed_flights = self.ed_flights()
+
+        with pytest.raises(
```
Comment (sethmlarson) [inline tests/dataframe/test_drop_pytest.py:64]: You need to have a separate `with pytest.raises(...)` for every statement you're expecting an error from.

File: tests/dataframe/test_drop_pytest.py [hunk 2]
```diff
@@ -15,17 +15,24 @@
...
+
 # File called _pytest for PyCharm compatability
+from tests.common import TestData
 
 
-class TestDataFrameDrop:
+class TestDataFrameDrop(TestData):
     def test_drop(self, df):
-        df.drop(["Carrier", "DestCityName"], axis=1)
+        df.drop(labels=["Carrier", "DestCityName"], axis=1)
```
Comment (sethmlarson) [inline tests/dataframe/test_drop_pytest.py:26]: Can we leave at least one without `labels=`.

File: tests/dataframe/test_drop_pytest.py [hunk 3]
```diff
@@ -15,17 +15,24 @@
...
 
-        df.drop(["1", "2"])
-        df.drop(["1", "2"], axis=0)
+        df.drop(labels=["1", "2"])
+        df.drop(labels=["1", "2"], axis=0)
         df.drop(index=["1", "2"])
+        df.drop(labels="3", axis=0)
+        df.drop(columns="Carrier")
+        df.drop(columns=["Carrier", "Carrier_1"], errors="ignore")
+        df.drop(columns=["Carrier_1"], errors="ignore")
```
Comment (sethmlarson) [inline tests/dataframe/test_drop_pytest.py:36]: Can we add an `assert` at the end of this test.

File: tests/dataframe/test_repr_pytest.py [hunk 4]
```diff
@@ -241,6 +241,14 @@ def test_num_rows_repr_html(self):
         # Restore default
         pd.set_option("display.show_dimensions", show_dimensions)
 
+    def test_num_rows_repr_html_display_none(self):
+        display = pd.get_option("display.notebook_repr_html")
```
Comment (sethmlarson) [inline tests/dataframe/test_repr_pytest.py:245]: Put this whole test in a `try:` after `display = ...` and the "restore default" step within a `finally:`

**Review by sethmlarson — CHANGES_REQUESTED** [review]
Looks pretty good, thanks! Few comments for you:

File: tests/dataframe/test_drop_pytest.py (hunk 2 above)
Comment (V1NAY8) [reply tests/dataframe/test_drop_pytest.py:26]: Yes

File: tests/dataframe/test_drop_pytest.py (hunk 3 above)
Comment (V1NAY8) [reply tests/dataframe/test_drop_pytest.py:36]: So `conftest.py` takes care of it. It takes the df operation and applies it to both pandas and eland then asserts it. In this method. 
https://github.com/elastic/eland/blob/f241ae971aeb332c05c77fa3321b70175ea057e8/tests/conftest.py#L88

Am I missing something here ?

File: tests/dataframe/test_repr_pytest.py (hunk 4 above)
Comment (V1NAY8) [reply tests/dataframe/test_repr_pytest.py:245]: Yes :) there are some more places, I added them too

**Comment by sethmlarson** [comment]
jenkins test this please

**Review by sethmlarson — APPROVED** [review]
LGTM, thanks for this!!

**Pull request merged**